src/openforms/utils/tests/test_migrations.py

```diff
@@ -13,13 +13,14 @@ class TestMigrations(TestCase):
     app = None
     migrate_from = None
     migrate_to = None
+    extra_from_migrations = []
 
     def setUp(self):
         assert self.migrate_from and self.migrate_to and self.app, (
             "TestCase '%s' must define migrate_from, migrate_to and app properties"
             % type(self).__name__
         )
-        self.migrate_from = [(self.app, self.migrate_from)]
+        self.migrate_from = [(self.app, self.migrate_from)] + self.extra_from_migrations
```

Comment:
I'm not sure about this, but it seems that when migrating to `migrate_from` the database state is properly migrated, but the models in the apps registry (retrieved in line 26) apparently not, causing issues: https://github.com/open-formulieren/open-forms/pull/2261/files#diff-c301558f7c67e1e1cd54331d556c2a6f4476a25ca524a5ac81d81a720b3e8245R26

Reply:
I made some changes to this in effb84f2750a67a1be9d1c1d88cdc4f173597fb8, so I'd like to see if that resolves it.

Reply:
Works!